src/Microsoft.DotNet.TemplateLocator/TemplateLocator.cs

```diff
@@ -54,7 +54,12 @@ public TemplateLocator()
                     nameof(dotnetRootPath));
             }
 
-            _workloadManifestProvider ??= new SdkDirectoryWorkloadManifestProvider(dotnetRootPath, sdkVersion, userProfileDir);
+            //  Will the current directory correspond to the folder we are creating a project in?  If we need
```

Comment:
Was this resolved? If not : is there an issue with this marked, so we can make sure we don't lose track of this TODO?